Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -66,12 +70,27 @@ public Initializer(string id)
                 : this(id, GoogleAuthConsts.TokenUrl) { }
 
             /// <summary>Constructs a new initializer using the given id and the token server URL.</summary>
-            public Initializer(string id, string tokenServerUrl) : base(tokenServerUrl)
+            public Initializer(string id, string tokenServerUrl)

```

Comment:
[optional] Since this change is so huge, I would prefer to avoid changes like this, I would just leave it as it is.